### PR TITLE
[Logs]: Fixed issue with missing tags when collecting logs from docker container

### PR DESCRIPTION
**Pull request by ajacquemot**
### What does this PR do?

Compute `ddtags` even if there is no tags specified in the integration file

### Motivation

Be able to collect docker tags all the time

### CODE REVIEW

File: pkg/logs/message/origin.go [hunk 1]
```diff
@@ -49,10 +49,12 @@ func (o *Origin) TagsPayload() []byte {
...
 		tagsPayload = append(tagsPayload, []byte("[dd ddsourcecategory=\""+o.LogSource.Config.SourceCategory+"\"]")...)
 	}
-	if o.LogSource.Config.Tags != "" {
+	if o.LogSource.Config.Tags != "" || len(o.tags) != 0 {
 		tags := o.LogSource.Config.Tags
-		if len(o.tags) > 0 {
+		if len(o.tags) > 0 && tags != "" {
 			tags = tags + "," + strings.Join(o.tags, ",")
+		} else {
+			tags = strings.Join(o.tags, ",")
```
Comment (mnshdw) [inline pkg/logs/message/origin.go]: This probably should not be called if len(o.tags) == 0.
In terms of readability, what do you think of have separate conditions ? ie
```
var tags string
o.LogSource.Config.Tags != "" {
  ...
}
if len(o.tags) > 0 {
  ...
}
```

**Review by mnshdw — COMMENTED** [review]
I'm OK with the change but I'm sure you can make the code easier to read.

File: pkg/logs/message/origin.go [hunk 2]
```diff
@@ -49,11 +49,18 @@ func (o *Origin) TagsPayload() []byte {
...
-		tags := o.LogSource.Config.Tags
-		if len(o.tags) > 0 {
+		tags = o.LogSource.Config.Tags
+	}
+	if len(o.tags) > 0 {
+		if tags != "" {
 			tags = tags + "," + strings.Join(o.tags, ",")
+		} else {
+			tags = strings.Join(o.tags, ",")
 		}
```
Comment (tmichelet) [inline pkg/logs/message/origin.go]: what about 
```
if tags != "" {
    tags = tags + ","
}
tags = tags + strings.Join(o.tags, ",")
```

Comment (tmichelet) [reply pkg/logs/message/origin.go]: Alternative would be to define `all_tags` as `[o.LogSource.Config.Tags, strings.Join(o.tags, ",")]` and do a join there

**Review by mnshdw — APPROVED** [review]

**Pull request merged**